### PR TITLE
Cleanup

### PR DESCRIPTION
**Pull request by jesse-wei**
- Switched to PySide6 because PyQt6 doesn't have documentation. There are only minor differences, but it's annoying to see something in PySide6 documentation and have it be named differently in PyQt6.
- The biggest difference is that you now need to do `./compileui.py` to compile the .ui file to .py.
  - Pros: We now don't get warnings for self.ui.action_open or whatever label, unlike before
  - Cons: Annoying to have to compile. Also, I could not figure out window switching with PySide6 earlier, but that's no longer an issue.
  - It's possible to load directly like before: https://doc.qt.io/qtforpython/tutorials/basictutorial/uifiles.html#option-b-loading-it-directly. Not 100% sure how to make that work yet. Specifically, looking at the documentation and [StackOverflow](https://stackoverflow.com/questions/7144313/loading-qtdesigners-ui-files-in-pyside), the syntax seems to be `window = load(ui_file)`, which would overwrite our `MainWindow` methods.
- Cleaned up `enable_elements` and `disable_elements` so that we don't have to modify them when adding a new GUI element except in edge cases.

### CODE REVIEW

File: compileui.py [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+"""This script runs a command that compiles the .ui file to a .py file.
+
+Must be run from .../HeadCircumferenceTool
+
+You must have PySide6 and the pyside6-uic command installed"""
+
+import os
+from src.utils.constants import UI_FILE_PATH, COMPILED_UI_FILE_PATH
+
+os.system(f"pyside6-uic {UI_FILE_PATH} > {COMPILED_UI_FILE_PATH}")
```
Comment (eric-unc) [inline compileui.py:12]: Are Python build utilities really so bad that we need this? More proof of Ruby superiority.

**Comment by eric-unc** [comment]
interesting

**Review by madiali — APPROVED** [review]
Nice work!

File: compileui.py (hunk 1 above)
Comment (jesse-wei) [reply compileui.py:12]: It's quite unfortunate. The syntax of PyQt6 was this

```py
class MainWindow
    loadUiFile(into MainWindow)
    # keep all of MainWindow's methods
```

Unfortunately, PySide6 syntax (for directly loading a .ui file) is like this

```
MainWindow = loadUi(ui file)
```

There does not seem to be a way around that in PySide6. Someone implemented the PyQt6 syntax in PySide (https://stackoverflow.com/questions/14892713/how-do-you-load-ui-files-onto-python-classes-with-pyside/14894550#14894550) but that's code from a long time ago. Did not get it to work.

**Pull request merged**